services/scan/src/build_cast_vote_record.ts

```diff
@@ -290,19 +276,42 @@ function buildCastVoteRecordFromHmpbPage(
       back.interpretation.metadata.pageNumber,
     ],
     ...votesEntries,
-    _ballotImages: hasWriteIns ? ballotImages : undefined,
     _layouts: ballotLayouts,
   };
 }
 
+export function addBallotImagesToCvr(cvr: CastVoteRecord, ballotImages: SheetOf<InlineBallotImage>): CastVoteRecord {
+  return {
+    ...cvr,
+    _ballotImages: ballotImages,
+  }
+}
+
+
+// returns booleans for front and back --> for now can only do true,true or false,false
+export function cvrHasWriteIns(election: Election, cvr: CastVoteRecord): [boolean, boolean] {
```

Comment:
Pass the contest IDs? The store knows how to get the contest IDs for a sheet being exported.